content/docs/for-platform-operators/how-to/build-inputs/build-env.md

```diff
@@ -0,0 +1,18 @@
++++
+title="Specify the build time environment variables"
+weight=4
++++
+
+Environment variables are any values used to allow `Buildpacks` configurability. Some environment variables can't be modified while others are expected to get changed to allow a level of customization.
```

Comment:
```suggestion
Environment variables are used to configure buildpack behavior. They may be specified by:
* The platform operator (this page)
* The end user (see XXX)
* Other buildpacks (see XXX)
```

When more than one entity specifies the same environment variable, the order of precedence is as shown above, with the platform operator having ultimate say over what the final value of the variable will be.

Reply:
We should fill in XXX with the right link

Reply:
updated